src/jobs/pages/designer-i-oslo.md

```diff
@@ -1,26 +1,24 @@
 ---
-title: Seniordesigner
-h1_title: Erfaren designer med entreprenørånd
+title: Designer i Oslo
+h1_title: En variant av en designer
 slug: erfaren-designer-i-oslo
-contact_emails: te@variant.no, mk@variant.no
-meta_title: Vi søker erfaren designer til Variant Oslo!
-meta_description: Vi ser etter en dyktig og erfaren designer med entreprenørånd som kan være med å forme et nytt selskap.
+contact_emails: lel@variant.no
+meta_title: Vil du være med oss å bygge Variant Oslo?
+meta_description: Vi ser etter en dyktig designer som vil være med å bygge Variant Oslo!
 meta_image: https://www.variant.no/images/erfaren_designer_meta.jpg
```

Comment:
Burde potensielt oppdatere meta-bilder også

Reply:
enig. Håper vi får litt bilder på plass for oslo snart og